src/main/java/lotto/controller/LottoController.java

```diff
@@ -0,0 +1,99 @@
+package lotto.controller;
+
+import lotto.domain.*;
+import lotto.domain.ticketfactory.FixedNumberTicketFactory;
+import lotto.exception.LottoCustomException;
+import lotto.utils.ValidateUtils;
+import lotto.view.InputView;
+import lotto.view.OutputView;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class LottoController {
+    private static final int EXIST = 1;
+    private static final String SEPARATOR = ",";
+    private final InputView inputView;
+
+    public LottoController(final Scanner scanner) {
+        inputView = new InputView(scanner);
+    }
+
+    public void run() {
+        Money money = inputMoney();
+        LottoTickets lottoTickets = buyLottoTickets(money);
+
+        LottoTicket winningTicket = makeWinningTicket();
+        LottoNumber bonusBall = makeBonusNumber(winningTicket);
+
+        showResult(money, lottoTickets, winningTicket, bonusBall);
+        ifChangeExist(money);
+    }
+
+    private void ifChangeExist(Money money) {
+        if (money.getChange() >= EXIST) {
+            OutputView.printGiveChange(money.getChange());
+        }
+    }
+
+    private Money inputMoney() {
+        try {
+            OutputView.printMoneyMessage();
+            Money money = new Money(ValidateUtils.parseInt(inputView.inputValue()));
```

Comment:
```suggestion
            Money money = new Money(inputView.inputValue());
```
1. 취향의 문제일 수는 있는데요, 이 정도 검증은 `InputView`에서 자체적으로 처리해도 되지 않을까요? 그렇게 하면 위에 보이는 것처럼 코드가 꽤 간단해질 것 같네요.
2. `inputValue()`라는 메서드 이름에 관해 좀 더 고민해 볼까요?

Reply:
이 부분은 페어와 저도 고민을 했었는데요, InputView에서 validation까지 해도 괜찮은지, 아니면 이번 프로젝트에서는 validate하는 부분이 단순 parseInt 정도여서 InputView에서 처리해도 되는 것인지 궁금합니다! 또한 inputValue()도 scanner.nextLine()이 공통적으로 들어가서 그냥 inputMoney, inputWinningNumbers로 분리하기 보다는 하나로 합치자!해서 inputValue()로 메서드 명을 지었었습니다. 하지만 만약 validation을 거친 값을 리턴할 수 있도록 변경된다면 각각을 구분지어서 명명할 수 있을 것 같아요.

Reply:
저는 문자열 파싱이나 숫자로의 변환 정도는 `InputView`에서 처리해도 된다고 생각해요. 그렇게 하면 도메인 객체들은 그런 잡다한 파싱 책임을 떠안지 않아도 되고, 자신들의 핵심 로직에 집중할 수 있거든요. 이건 저의 생각일 뿐이니까 조앤도 여러 방법들을 다 시도해 보고 무엇이 더 좋게 느껴지는지 경험했으면 좋겠어요!